plugins/connection/httpapi.py

```diff
@@ -6,8 +6,8 @@
 __metaclass__ = type
 
 DOCUMENTATION = """
-author: fischbacher.markus@gmail.com
-connection: zabbix
+name: zabbix
```

Comment:
```suggestion
name: httpapi
```
The plugin is named `httpapi`, not `zabbix`. See for example `ansible-doc -t connection -l community.zabbix`.